rca/people/migrations/0001_initial.py

```diff
@@ -99,19 +274,33 @@ class Migration(migrations.Migration):
                         verbose_name="ID",
                     ),
                 ),
-                ("phone_number", models.CharField(max_length=255)),
+                (
+                    "sort_order",
+                    models.IntegerField(blank=True, editable=False, null=True),
+                ),
                 (
                     "page",
+                    models.ForeignKey(
+                        blank=True,
+                        null=True,
+                        on_delete=django.db.models.deletion.CASCADE,
+                        related_name="+",
+                        to="wagtailcore.Page",
+                    ),
+                ),
+                (
+                    "source_page",
                     modelcluster.fields.ParentalKey(
                         on_delete=django.db.models.deletion.CASCADE,
-                        related_name="phone_numbers",
+                        related_name="related_school_pages",
                         to="people.StaffPage",
                     ),
                 ),
             ],
+            options={"ordering": ["sort_order"], "abstract": False},
         ),
         migrations.CreateModel(
-            name="SocialMediaProfile",
```

Comment:
I'm not sure overwriting the initial migration is the right approach here. Surely you want a new migration to remove the old models, followed by another to add the new ones? Otherwise you end up with old tables sticking around in your database.

Reply:
I'll need to reset the DB on dev I think now though